integrations/grpc.go

```diff
@@ -178,10 +176,15 @@ func (s *tracedClientStream) RecvMsg(m interface{}) error {
 	return err
 }
 
+// WithClientUnaryInterceptor function adds unary client interceptor to store its request, 
+// response and external dependencies. It should be called in grpc.Dial method.
```

Comment:
give an example of how this will be called. because many users may not be familiar with the interceptor.